docs/source/installation.md

```diff
@@ -1,49 +1,67 @@
 # Installation
 
-To use **pyCoilGen**, it needs to installed.
+> **Note:** Ensure that you have the necessary permissions to install packages on your system. Consider using a [Python virtual environment](https://docs.python.org/3/library/venv.html) to manage your dependencies if necessary.
+
+To use **pyCoilGen**, you need to:
+
+* [install a supported version of Python](install-python)
+* [install the `pycoilgen` package](install-pycoilgen)
+* [install other dependencies](other-dependencies-blas-and-gfortran), if these are not already on your system.
+
+Optionally, you can install:
+
+* the [**pyCoilGen** data package](install-pycoilgen-data-package-optional)
+* [FastHenry2](fasthenry2)
+
 
 ## Install Python
 
-**pyCoilGen** depends on Python >= 3.6. It has been tested with Python 3.7.3 and 3.9.2.
+**pyCoilGen** depends on Python >= 3.6. 
 
 Please follow the instructions for your operating system to [install Python](https://www.python.org/downloads/).
 
-## Installing pyCoilGen
-
-**Note:** Ensure that you have the necessary permissions to install packages on your system. Consider using a [virtual environment](https://docs.python.org/3/library/venv.html) to manage your dependencies if necessary.
+## Install pyCoilGen
 
 To install **pyCoilGen**, you can use `pip`, the Python package manager. 
 
 ```bash
 $ pip install pycoilgen
 ```
 
-## Optional Extras
-There is an optional data package for **pyCoilGen** that provides coil mesh surface `.stl` files, pre-calculated target fields and solutions that can also be installed using `pip`.
+## Other Dependencies - BLAS and gfortran
+
+You may also need to manually install [BLAS](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms). On some Linux systems, BLAS also depends on gfortran.
 
 ```bash
-$ pip install pycoilgen_data 
+$ sudo apt-get install libopenblas-dev gfortran
 ```
 
-This will install the specified extras along with the main package. These files will automatically by detected by **pyCoilGen** after the **pyCoilGenData** package has been installed.
+## Optional packages
 
+### Install pyCoilGen Data Package
 
-## SciPiy and Dependencies
+There is an optional data package for **pyCoilGen** that provides coil mesh surface `.stl` files, pre-calculated target fields and solutions. This package can be installed with `pip`.
```

Comment:
Add more details about what this package contains

Reply:
Addressed with commit #1f11492